ChangeLog.md

```diff
@@ -3,12 +3,20 @@
 
 * __next version__ - XXXX-XX-XX
 
+  * Add feedback for correct answers in submitted answer panel (Brian Mercer).
+
   * Add Learning Tools Interoperability LTI 1.1.1 tool provider functionality (Dave Mussulman).
 
   * Add course instance admin subpages (Dave Mussulman).
 
   * Add networkx Python library to plbase (Nathan Bowman).
 
+  * Add option to toggle placeholder help text for `pl-number-input` (James Balamuta).
+
+  * Add `size` attribute to `pl-number-input` (Nicolas Nytko).
+
+  * Add demo question showcasing all options for `pl-number-input` (James Balamuta).
```

Comment:
@nicknytko please add yourself to all these ChangeLog entries -- we want to credit you! You can say "James Balamuta and Nicolas Nytko" if you want.